tests/test-sources/plugins/git/octo.nix

```diff
@@ -0,0 +1,23 @@
+{
+  empty = {
+    plugins.octo.enable = true;
+  };
+
```

Comment:
Add a `default` test case as well.

Reply:
Based on the options you've declared, I think the default test would look like this:

```nix
defaults = {
  plugins.octo.enable = true;

  settings = {
    use_local_fs = false;
    enable_builtin = false;
    reaction_viewer_hint_icon = "";
    user_icon = " ";
    timeline_marker = "";
    timeline_indent = "2";
    right_bubble_delimiter = "";
    left_bubble_delimiter = "";
    github_hostname = "";
    snippet_context_lines = 4;
    timeout = 5000;
    ui.use_sign_column = true;

    default_remote = [ "upstream" "origin" ];

    gh_env = { };
    ssh_aliases = { };

    picker_config = {
      use_emojis = false;

      mappings = {
        open_in_browser.lhs = "<C-b>";
        copy_url.lhs = "<C-y>";
        checkout_pr.lhs = "<C-o>";
        merge_pr.lhs = "<C-r>";
      };
    };

    issues.order_by = {
      field = "CREATED_AT";
      direction = "DESC";
    };
  };
};
```